shared/hsm.py

```diff
@@ -759,7 +759,10 @@ async def approve_transaction(self, psbt, psbt_sha, story):
                 for idx, tx_out in psbt.output_iter():
                     if not psbt.outputs[idx].is_change:
                         total_out += tx_out.nValue
-                        dests.append(chain.render_address(tx_out.scriptPubKey))
+                        try:
+                            dests.append(chain.render_address(tx_out.scriptPubKey))
+                        except ValueError:
+                            dests.append(str(b2a_hex(tx_out.scriptPubKey), 'ascii'))
```

Comment:
This must render OP_RETURN and raise warning if we can't understand the destination. [Maybe a policy flag (default off) that permits sending to unknown scripts—but no one is asking for that.]
We need to refactor `auth.py: render_output()` into either `psbt.py` or `chains.py`

Reply:
Hmmm.. altho this is just for the policy engine. IDK